test/utils/math/Math.test.js

```diff
@@ -85,4 +87,94 @@ contract('Math', function (accounts) {
       expect(await this.math.ceilDiv(MAX_UINT256, b)).to.be.bignumber.equal(MAX_UINT256);
     });
   });
+
+  describe('muldiv', function () {
```

Comment:
Please refactor these tests so that there's a single describe block for "rounds down", and the same for "rounds up". There's currently two separate ones.